apps/designer/app/canvas/features/wrapper-component/wrapper-component.tsx

```diff
@@ -40,19 +38,18 @@ const ContentEditable = ({
 
 type WrapperComponentDevProps = {
   instance: Instance;
-  css: CSS;
   children: Array<JSX.Element | string>;
   onChangeChildren?: OnChangeChildren;
 };
 
 export const WrapperComponentDev = ({
   instance,
-  css,
   children,
-  onChangeChildren = noop,
+  onChangeChildren,
   ...rest
 }: WrapperComponentDevProps) => {
-  const className = useCss({ instance, css });
+  useCssRules(instance);
```

Comment:
We could decouple this from react and for example watch any changes in tree.
Just a thought

Reply:
yep yep, also wasn't sure what to take. This would remove the need to rerender the react tree entirely on style changes, not only for live updates.

Reply:
I kept it because I wasn't sure how to decouple it while making sure the styles for the new component are rendered before the component HTML when you create a new one.
We would probably still need to have some hook to hook into layout effect of the component